modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestClusterTest.java

```diff
@@ -45,6 +46,12 @@ public class TestClusterTest {
             .addColumn("VAL", NativeTypes.stringOf(64))
             .defaultDataProvider(dataProvider)
             .end()
+            .addHashIndex()
```

Comment:
It would be better to make index builder as child of table builder: 

```
TestBuilders.cluster()
    .nodes("N1")
    .addTable()
        .name("T1")
        .distribution(IgniteDistributions.hash(List.of(0)))
            .addColumn("ID", NativeTypes.INT32)
            .addColumn("VAL", NativeTypes.stringOf(64))
            .defaultDataProvider(dataProvider)
            .addHashIndex()
                .name("IDX_ID")
                .addColumn("ID")
                .defaultDataProvider(dataProvider)
            .end()
        .end()
```

Reply:
fixed

Reply:
You've changed indexes in cluster builder flow, but leave top level builder indexes flow as is. Let's make it consistent